integration-libs/punchout/components/punchout-buttons/punchout-buttons.component.html

```diff
@@ -11,7 +12,9 @@
     type="button"
     class="btn btn-block btn-primary"
     (click)="submitRequisition(true)"
+    [disabled]="closeSessionTriggered"
   >
     {{ 'punchout.cancel' | cxTranslate }}
   </button>
 </ng-container>
+<cx-punchout-requsition *ngIf="closeSessionTriggered" />
```

Comment:
typo from early days: cx-punchout-requsition to cx-punchout-requisition
can be changed later too

Reply:
Will it not be a breaking change if we rename it?